From 3d65b3eb625fe14e3aa3bad26f659e20aee477b3 Mon Sep 17 00:00:00 2001
From: Charles-IV <cthefourth@gmail.com>
Date: Wed, 11 Jul 2018 12:56:27 +0100
Subject: [PATCH 209/249] wiki: Refine install template

Userdata must be formatted to ensure stock encryption is removed.
This is necessary for A/B devices as Lineage recovery does not decrypt
internal storage.
Cache partition also does not exist on A/B devices.

Change-Id: Iabe2a371695b40e0fc785b7a566d7ebc52e91ac2
---
 _includes/templates/device_install.md | 32 ++++++++++++++++-----------
 1 file changed, 19 insertions(+), 13 deletions(-)

diff --git a/_includes/templates/device_install.md b/_includes/templates/device_install.md
index d6257f0..7c6e395 100644
--- a/_includes/templates/device_install.md
+++ b/_includes/templates/device_install.md
@@ -47,26 +47,32 @@ There are no recovery install instructions for this discontinued device.
 1. [Build]({{ "devices/" | append: device.codename | append: "/build" | relative_url }}) a LineageOS install package.
 {% endif %}
     * Optionally, download 3rd party application packages such as [Google Apps]({{ "gapps.html" | relative_url }}) (use the `{{ userspace_architecture }}` architecture)
-2. Place the LineageOS `.zip` package, as well as any other .zip packages on the root of `/sdcard`:
+2. If you aren't already in recovery, reboot into recovery:
+    * {{ device.recovery_boot }}
+3. _(Optional, but recommended)_: Tap the **Backup** button to create a backup. Make sure {% if device.sdcard != blank %}the backup is created in the external sdcard or copy it{% else %}the backup is copied{% endif %} onto your computer as the internal storage will be formatted.
+4. Go back to return to main menu, then tap **Wipe**.
+5. Now tap **Format Data** and continue with the formatting process. This will remove encryption as well as delete all files stored on the internal storage.
+6. Return to the previous menu and tap **Advanced Wipe**.
+{% if device.is_ab_device %}
+7. Select the *System* partition to be wiped and then **Swipe to Wipe**.
+{% else %}
+7. Select the *Cache* and *System* partitions to be wiped and then **Swipe to Wipe**.
+{% endif %}
+8. Place the LineageOS `.zip` package, as well as any other .zip packages on the root of `/sdcard`:
     * Using adb: `adb push filename.zip /sdcard/`
     * You can use any method you are comfortable with. `adb` is universal across all devices, and works both in Android and recovery mode, providing
         USB debugging is enabled.
-3. If you aren't already in recovery, reboot into recovery:
-    * {{ device.recovery_boot }}
-4. _(Optional, but recommended)_: Select the **Backup** button to create a backup.
-5. Select **Wipe** and then **Advanced Wipe**.
-6. Select *Cache*, *System* and *Data* partitions to be wiped and then **Swipe to Wipe**.
-7. Go back to return to main menu, then select **Install**.
-8. Navigate to `/sdcard`, and select the LineageOS `.zip` package.
-9. Follow the on-screen prompts to install the package.
+9. Go back to return to main menu, then tap **Install**.
+10. Navigate to `/sdcard`, and select the LineageOS `.zip` package.
+11. Follow the on-screen prompts to install the package.
 {% if device.is_ab_device %}
-10. _(Optional)_: Reboot to recovery and install any additional packages using the same method. The reboot is necessary on devices with A/B (seamless) system updates.
+12. _(Optional)_: Reboot to recovery and install any additional packages using the same method. The reboot is necessary on devices with A/B (seamless) system updates.
 {% else %}
-10. _(Optional)_: Install any additional packages using the same method.
+13. _(Optional)_: Install any additional packages using the same method.
 {% endif %}
     {% include note.html content="If you want any Google Apps on your device, you must follow this step **before** the first boot into Android!" %}
-11. _(Optional)_: Root the device by installing the [LineageOS su add-on](https://download.lineageos.org/extras) (use the `{{ userspace_architecture }}` package) or using any other method you prefer.
-12. Once installation has finished, return to the main menu, select **Reboot**, and then **System**.
+14. _(Optional)_: Root the device by installing the [LineageOS su add-on](https://download.lineageos.org/extras) (use the `{{ userspace_architecture }}` package) or using any other method you prefer.
+15. Once installation has finished, return to the main menu, tap **Reboot**, and then **System**.
 
 ## Get assistance
 
-- 
2.17.1

